python/SocketMessages.py

```diff
@@ -128,10 +128,17 @@ def __init__(self, script_text: list[str], inspection_points: list[dict]):
 
         for point in sorted_inspection_points:
             self.inspectionPoints.append(
-                InspectionPointFormatFromFrontend(point["id"], point["lineNumber"] - 1, point["command"])
+                InspectionPointFormatFromFrontend(
+                    point["id"],
+                    point["lineNumber"] - 1,
+                    point["command"],
+                    [InspectionVariable(var["name"], var["readCommand"]) for var in point["additionalVariablesToRead"]]
+                )
             )
 
-
+        for globalVariable in globalVariables:
+            parsed = InspectionVariable(globalVariable["name"], globalVariable["readCommand"])
+            self.globalVariables.append(parsed)
```

Comment:
I don't quite understand why we have to loop through globalvariables to assign a self.globalvariables?